docs/Content structure/Paragraphs/Class Location.md

```diff
@@ -0,0 +1,6 @@
+# Class Location
+
+This is dynamic paragraph that renders the locations class location view mode, based on url query parameter location with a valid id.
+
+### Fields
+NONE
```

Comment:
Actually it has field in the structure - https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Content%20structure/Paragraphs/Latest%20Blog%20Posts%20(Camp).md#fields.